src/utils/commands.js

```diff
@@ -35,6 +35,7 @@ const replCommands = [
   'workbench',
   'nim',
   'command_set',
+  'history',
```

Comment:
We should probably disallow `command_create` to create these restricted commands to avoid confusion. Commander can create these commands but the user will not be able to run them. Can you create an issue for this?

Reply:
Done.